libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -9,7 +9,6 @@
 #include <GCS_MAVLink/GCS.h>
 #include <SRV_Channel/SRV_Channel.h>
 #include <AP_Logger/AP_Logger.h>
-#include <AP_GPS/AP_GPS.h>
```

Comment:
This is wrong, GPS is needed, you are getting it transitively right now, but removing it isn't actually correct.

Reply:
removed